src/internal_events/tcp.rs

```diff
@@ -0,0 +1,135 @@
+use super::InternalEvent;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct TcpConnectionEstablished {
+    pub peer_addr: Option<std::net::SocketAddr>,
+}
+
+impl InternalEvent for TcpConnectionEstablished {
+    fn emit_logs(&self) {
+        if let Some(peer_addr) = self.peer_addr {
+            debug!(message = "connected", %peer_addr);
```

Comment:
```suggestion
            debug!(message = "connected.", %peer_addr);
```

Reply:
I'm still not sold on these periods :smile: 

We'll need to make another pass to get all our logging aligned, so I'm leaving these as they are for now.

Reply:
Only saying this because the other logs have it :D